challenge.js

```diff
@@ -10,6 +10,8 @@
 
 // ANSWER
 
+  /** You would add an event handler to some item on the page that when executed, invokes a method on the 'document' object that produces the desired element. The method for adding an html element is: document.createElement() **/
```

Comment:
We do frequently alter DOM elements or attach new DOM elements from event handlers, but it is not necessary that we do these operations from within an event handler. `document.createElement` is a very useful `document` method that creates a new HTML element object. However, it does not attach it to the DOM -- if it did, where would it attach it? Most likely in the wrong place! What it does is return a reference to the object to you, for use as a utility or for attaching to the DOM in the proper location. 3/4